docs/source/modules.rst

```diff
@@ -172,6 +172,11 @@ analyses/seasidecge
 ..  autoclass:: seasidecge.solverconstants.equationlib.ExprM
     :members:
 
+analyses/socialvulnerability
+===================
```

Comment:
Sphinx needs == to underscore the whole string, the same number of characters
analyses/socialvulnerability
===================